examples/kafka/kafka-ephemeral.yaml

```diff
@@ -20,6 +21,4 @@ spec:
     replicas: 3
     storage:
       type: ephemeral
-  entityOperator:
-    topicOperator: {}
-    userOperator: {}
+  topicOperator: {}
```

Comment:
I think this should not be here!

Reply:
@tombentley Your last commit removed the `topicOperator`. But is didn't revert the removal of the `entityOperator`. We have to add that back.